src/main/java/com/exasol/adapter/dialects/postgresql/PostgresSQLSqlGenerationVisitor.java

```diff
@@ -87,8 +88,8 @@ private String buildColumnProjectionString(final SqlColumn column, final String
             }
             return false;
         } catch (final AdapterException exception) {
-            throw new SqlGenerationVisitorException("Exception during deserialization of ColumnAdapterNotes. ",
-                    exception);
+            throw new SqlGenerationVisitorException(ExaError.messageBuilder("E-PGVS-7")
```

Comment:
Maybe this is a Fatal, not an error? I think also "Create a GitHub issue" mitigation fits here

Reply:
As this is going to be fixed in the Parser directly, and this we'll be removed, for now I'll leave it as it is